tests/test_models/test_backbones.py

```diff
@@ -296,3 +296,29 @@ def test_pointnet2_sa_msg():
     assert sa_indices[2].shape == torch.Size([1, 256])
     assert sa_indices[3].shape == torch.Size([1, 64])
     assert sa_indices[4].shape == torch.Size([1, 16])
+
+
+def test_dla_net():
+
+    if not torch.cuda.is_available():
```

Comment:
Is GPU necessary for `DLANet`? From the code I didn't see any CUDA specific ops, so I guess it can also run on CPU? Or maybe the DeformConv requires CUDA?